app/grants/clr.py

```diff
@@ -587,3 +611,117 @@ def predict_clr_live(grant, contributor, amount, is_postive_vote=True):
     )
 
     return predicted_clr_match
+
+
+'''
+    Server Side Grants Transaction Validator
+
+    Args:
+        list_contributions : csv | text | file
+
+    Returns:
+        status: sucess or fail, reason
+        sucess_validator: [{'sucess': <int>, 'reason': <Object>}]
+        fail_validator: [{'fail': <int>, 'reason': <Object>}]
+
+'''
+
+def grants_transaction_validator(list_contributions):
+    """This function check grants transaction list"""
+    if isinstance(list_contributions, list):
+        df = pd.DataFrame(list_contributions=list_contributions[1:,1:],index=list_contributions[1:,0], columns=list_contributions[0,1:])
```

Comment:
```
Traceback (most recent call last):
  File "./manage.py", line 59, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.6/dist-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.6/dist-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.6/dist-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.6/dist-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/code/app/grants/management/commands/transaction_validator.py", line 27, in handle
    grants_transaction_validator(inputs)
  File "/code/app/grants/clr.py", line 630, in grants_transaction_validator
    df = pd.DataFrame(list_contributions=list_contributions[1:,1:],index=list_contributions[1:,0], columns=list_contributions[0,1:])
TypeError: list indices must be integers or slices, not tuple
```

Reply:
I need to transform the list into tuples
‘‘’tuple(list_contributions) ‘’’